### ISSUE TITLE
Error, no DATA reply

### ISSUE THREAD

**Issue by rgpublic** [munjeni/newflasher#72]
I always get "Error, no DATA reply". I tried time and again. The weird thing is: I already used this tool successfully two times. On this PC (Manjaro Linux). I even tried the same folder with the same flasher version (v57) again. For the life of me, I can't get it to work :-( I'm at a loss here. How can I debug this further? What could possibly be the reason for this. Manjaro is a rolling release Linux. Perhaps a new kernel, some permissions that got changed inadvertently?

![image](https://github.com/user-attachments/assets/9528b77c-1d30-498b-93e6-787431f43550)

**Comment by munjeni** [comment]
Rnning in virtual machine? If yes that it might be isue between your host machine and virtual machine usb access. If not I'm sorry I have no idea and to be honest time for Manjaro Linux

**Comment by rgpublic** [comment]
No, I'm not running on a virtual machine and I'm already running as root. Fastboot, ADB etc. all run finde. But sure, totally understand you can't support any arbitrary system. Just thought the error message itself might point to something. Nevertheless: Thanks a lot for your answer and for making this tool.

**Comment by Mittans** [comment]
I also run Manjaro and hit this error when trying to flash my Xperia 1 IV

**Comment by munjeni** [comment]
What say your dmesg when you hit error?

**Comment by munjeni** [comment]
Under Linux it work but under Manjaro what ever those distro is I have no idea sorry. The same for Mac and all the similar.
